js/app/game.js

```diff
@@ -290,6 +294,14 @@ define([
                 render: true,
             });
         }
+        var dropItem = (itemKey) => {
+            this.isValidItemKey(itemKey, true);
+            this.items[itemKey].owned = false;
+            this.mouse.updateCursor('default');
```

Comment:
why do you need this ?

Reply:
The rendering is doing that already, no ?

Reply:
The editor needs a way to remove items.